.unreleased/breaking-changes/2696-quint-deserialization.md

```diff
@@ -0,0 +1 @@
+- Update Quint deserialization for compatibility with version > 0.13.0, see #2696
```

Comment:
Itemization is added when we concatenate log entries :)
I forget which way it is all the time.

```suggestion
Update Quint deserialization for compatibility with version > 0.13.0, see #2696
```